npc/re/jobs/3-2/sura.txt

```diff
@@ -654,14 +654,14 @@ OnTimer305000:
 OnTimer315000:
 	mapannounce "sword_2-1","Come on here.",bc_map;
 	donpcevent "Buddy#job_shuaneh::OnDisable";
-	donpcevent "#Sura_garajjom::OnEnable";
+	// donpcevent "#Sura_garajjom::OnEnable";
 	end;
 OnTimer320000:
 	mapannounce "sword_2-1","I guess you're not ready to meet the master yet...",bc_map;
 	mapwarp "sword_2-1","yuno_fild07",255,178;
-	donpcevent "#Sura_garajjom::OnDisable";
-	donpcevent "Drawing Room::OnEnable";
+	// donpcevent "#Sura_garajjom::OnDisable";
```

Comment:
I tried to be the less invasive possible with the script, I comment the line instead of removing it because it is intended to be there (I did comparison with official script), but having it allows player to always success the test.

However, if should be removed, I can remove it without any problem :)